security/js-leaks.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Detect changes to JS objects in global scope</title>
+    <script src="./leaks.js"></script>
+    <link href='./style.css' rel='stylesheet'></link>
+</head>
+
+<body>
+    <p><a href="../">[Home]</a> ↣ <strong>[Detect changes to JS objects in global scope]</strong></p>
+
+    <p>
+        This page checks the global window object for differences versus standard browsers. 
+        This shows protections' footprint that are observable and potentially exploitable by the page.
+    </p>
+    <p>
+        To generate a profile for a new browser load this page there and export the <code>collectedProps</code> global. e.g. in most devtools you can write <code>copy(JSON.stringify(collectedProps))</code> to get the object on the clipboard.
```

Comment:
Nice! Let's also leave a note for the person implementing automatic test how to run this page automatically.